typescript/lib/parsers/openai.ts

```diff
@@ -381,6 +388,10 @@ export class OpenAIChatModelParser extends ParameterizedModelParser<Chat.ChatCom
     // Rename the last prompt to the requested prompt name
     prompts[prompts.length - 1].name = promptName;
 
+    const endEvent = new CallbackEvent("on_serialize_end", __filename, {
+      result: prompts,
```

Comment:
Same here r.e. `result` just being `prompts` key instead -- it doesn't seem like we are using a common `result` key for anything so using the more specific key makes more sense

Reply:
Revisiting the 'result' and the output data as a todo, but will probably get rid of the `result` as suggested